x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/components/actions_menu.tsx

```diff
@@ -108,8 +108,9 @@ export const AgentPolicyActionMenu = memo<{
 
           const manageAutoUpgradeAgentsItem = (
             <EuiContextMenuItem
+              data-test-subj="agentPolicyActionMenuManageAutoUpgradeAgentsButton"
               icon="gear"
-              disabled={!authz.fleet.allAgentPolicies}
+              disabled={!authz.fleet.allAgentPolicies || !authz.fleet.allAgents}
```

Comment:
nit: should we show a tooltip explaining user miss permissions, we are sometimes doing that, but not consistently through Fleet UI